src/components/HeroSection/JoinUs.jsx

```diff
@@ -0,0 +1,147 @@
+import React from 'react';
+import styled from 'styled-components';
+import tw from 'twin.macro';
+import { Body, ButtonLight, Container, Heading2 } from '..';
+import { join } from '../../../config/content';
+
+const Wrapper = styled.div`
+  ${tw`
+    items-center
+    justify-center
+    w-full
+    overflow-x-hidden
+    overflow-y-hidden
+    grid
+    xl:grid-cols-2
+    grid-cols-3
+    gap-6
+    mt-40
+    `}
+`;
+const Description = styled(Body)`
+  ${tw`
+    pt-5
+    pb-5
+    `}
+`;
+const JoinUsDiv = styled.div`
+  ${tw`
+    col-span-2
+    grid
+    grid-cols-3
+    bg-secondary
+    rounded-lg
+    w-full
+    h-full
+    lg:grid-cols-1
+    lg:mb-6
+    `}
+`;
+const JoinUsImg = styled.img`
+  ${tw`
+    block
+    col-span-2
+    w-auto
+    max-w-full
+    overflow-x-hidden
+    overflow-y-hidden
+    h-full
+    `}
+`;
+const JoinUsText = styled.div`
+  ${tw`
+    col-span-1
+    grid
+    p-5
+    h-full
+    justify-center
+    items-center
+    justify-items-center
+    `}
+  grid-template-rows: repeat(10, minmax(0, 1fr));
+`;
+const Head = styled(Body)`
+  ${tw`
+    row-span-1
+    `}
+`;
+const JoinHeading = styled(Heading2)`
+  ${tw`
+    row-span-3
+    lg:text-center
+    `}
+`;
+const SponsorHeading = styled(Heading2)`
+  ${tw`
+    row-span-2
+    lg:text-center
+    `}
+`;
+const SponsorDiv = styled.div`
+  ${tw`
+    xl:hidden
+    col-span-1
+    bg-secondary
+    rounded-lg
+    w-full
+    h-full
+    p-5
+    grid
+    justify-center
+    items-center
+    justify-items-center
+    `}
+  grid-template-rows: repeat(10, minmax(0, 1fr));
+`;
+
+const JoinDescription = styled(Description)`
+  ${tw`
+    row-span-4
+    justify-center
+    lg:text-center
+    `}
+`;
+const SponsorDescription = styled(Description)`
+  ${tw`
+    row-span-3
+    lg:text-center
+    `}
+`;
+const ButtonContainer = styled(ButtonLight)`
+  ${tw`
+    row-span-2
+    `}
+`;
+const SponsorImg = styled.img`
+  ${tw`
+    h-28
+    justify-items-center
+    row-span-3
+    `}
+`;
+
+function JoinUs() {
+  return (
+    <Container>
+      <Wrapper>
+        <JoinUsDiv>
+          <JoinUsImg src={join.JoinUs.img.src} alt={join.JoinUs.img.alt} />
+          <JoinUsText>
+            <Head>{join.JoinUs.head}</Head>
+            <JoinHeading>{join.JoinUs.heading}</JoinHeading>
+            <JoinDescription>{join.JoinUs.description}</JoinDescription>
+            <ButtonContainer>Join Us</ButtonContainer>
+          </JoinUsText>
+        </JoinUsDiv>
+        <SponsorDiv>
+          <SponsorHeading>{join.SponsorUs.heading}</SponsorHeading>
+          <SponsorDescription>{join.SponsorUs.description}</SponsorDescription>
+          <SponsorImg src={join.SponsorUs.img.src} alt={join.SponsorUs.img.alt} />
+          <ButtonContainer>Sponsor Us</ButtonContainer>
```

Comment:
Sponsor us button is not working

Reply:
You can find the links for both the typeforms in config/website.js

Reply:
@abhibhaw @nayakastha what do you think? should we embed Typeform in the website which pops up on clicking the buttons? or should we redirect them on other tab?

Reply:
I personally feel that we should embed them like a pop up

Reply:
embedding is better

Reply:
That will increase the page view time, better SEO :)